src/contribute/design-documents.md

```diff
@@ -123,4 +123,4 @@ This is a collection of documents that may or may not be up to date.
 ## Other
 Some of documents are available on this [google drive](https://drive.google.com/corp/drive/folders/0B-IhJZh9Ab52OFBVZHpsNjc4eXc)
 
-To add new design document it is recommended to use this [design document template](https://docs.google.com/document/d/1kVePqjt2daZd0bQHGUwghlcLbhvrny7VpflAzk9sjUg)
\ No newline at end of file
+To add new design document it is recommended to use this [design document template](https://docs.google.com/document/d/1kVePqjt2daZd0bQHGUwghlcLbhvrny7VpflAzk9sjUg)
```

Comment:
I don't see any changes here. Is it only padding/eol character or anything else that I missed?